src/resources/js/tickets-block.js

```diff
@@ -1328,9 +1469,10 @@ tribe.tickets.block = {
 
 			$.ajax({
 				type: 'POST',
-				url: obj.getRestEndpoint(),
+				url: obj.getRestEndpoint() + '/11/',
```

Comment:
Intentional?

Reply:
That was to demo the error - I should remove it from the PR though.